.github/workflows/main.yml

```diff
@@ -0,0 +1,51 @@
+name: lf-sample-repository-api-nodejs-CI
+
+on:
+  push:
+    branches: [ '\d+.x' ]
+  pull_request:
+    branches: [ '\d+.x' ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Echo build number
+        run: echo ${{github.run_id}}
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.5.1
+        with:
+          node-version: '14'
+
+      - name: Npm tool install
+        run: npm install -g npm@8
+
+      - name: Npm i
+        run: npm i
+
+      - name: Npm run lint
+        run: npm run lint
+      
+      - name: Npm run build
+        run: npm run build
+        
+      - name: Echo NPM package version
```

Comment:
probably don't need this step and the one below for creating a build artifact since those are used when creating a npm package in the client lib pipelines

Reply:
removed